tests/phpunit/tests/theme.php

```diff
@@ -309,19 +315,38 @@ public function test_switch_theme() {
 				$this->assertSame( $root_fs . '/' . get_template(), get_template_directory() );
 				$this->assertSame( $root_uri . '/' . get_template(), get_template_directory_uri() );
 
-				// get_query_template()
+				// Skip block themes for get_query_template() tests since this test is focused on classic templates.
+				if ( wp_is_block_theme() && current_theme_supports( 'block-templates' ) ) {
+					continue;
+				}
```

Comment:
@spacedmonkey See my reply in https://github.com/WordPress/wordpress-develop/pull/5160/files#r1331788580. We shouldn't add that here since this is a massive legacy test, written at a time where only classic themes existed. Most of what is tested here does not apply to block themes, and fixing this test to work for block themes is a waste of time.